projects/main_ping_delays_pingsender.kp/knowledge.md

```diff
@@ -0,0 +1,387 @@
+---
+title: Main Ping Submission and Recording Delays - pingSender
```

Comment:
Oh, and you say in your titles that you're looking at Recording Delay, but aren't.